src/main/java/seedu/command/EditContactCommand.java

```diff
@@ -67,6 +72,14 @@ private Boolean hasDuplicates(Contact postEditContact, ContactList contactList,
         return false;
     }
 
+    private Boolean[] hasEditedFieldChecker(String[] contactDetails) {
```

Comment:
Minor edit, apologies for being anal:
```suggestion
    private boolean[] hasEditedFields(String[] contactDetails) {
```